layouts/DocsLayout.tsx

```diff
@@ -1,20 +1,20 @@
 import BaseLayout from './BaseLayout';
 import SideNavigation from '../components/SideNavigation';
-import { useNodeData } from '../hooks/useNodeData';
+import { useNodeReleasesData } from '../hooks/useNodeReleasesData';
 import type { FC, PropsWithChildren } from 'react';
 
 const DocsLayout: FC<PropsWithChildren> = ({ children }) => {
-  const { currentLtsVersion, currentNodeVersion } = useNodeData();
+  const { lts, current } = useNodeReleasesData();
```

Comment:
As I mentioned before, both lts and current should not be nullable, they should always exist.